scripts/validate_response.py

```diff
@@ -97,12 +99,15 @@ def get_response_quality(self, args, qa_pairs):
             question = qa_pairs[query_id]["question"]
             answer = qa_pairs[query_id]["answer"]
 
-            response = requests.post(
-                # API question validator can be disabled.
-                "http://localhost:8080/v1/query",
+            response = api_client.post(
+                "/v1/query",
                 json={"query": question},
-                timeout=90,
-            ).json()["response"]
+                timeout=LLM_REST_API_TIMEOUT,
+            )
+            if response.status_code != 200:
```

Comment:
`requests` contains all HTTP code if you would like to avoid using magic constants